CI/integration_tests/transformations/test_molecular_mapping_results.py

```diff
@@ -0,0 +1,1269 @@
+                    "5": {"B": [5], "F": [20, 21, 22, 23]},
+                    "6": {"B": [6], "F": [24, 25, 26, 27]},
+                    "7": {"B": [7], "F": [28, 29, 30, 31]},
+                    "8": {"B": [8], "F": [32, 33, 34, 35]},
+                    "9": {"B": [9], "F": [36, 37, 38, 39]},
+                    "10": {"B": [10], "F": [40, 41, 42, 43]},
+                    "11": {"B": [11], "F": [44, 45, 46, 47]},
+                    "12": {"B": [12], "F": [48, 49, 50, 51]},
+                    "13": {"B": [13], "F": [52, 53, 54, 55]},
+                    "14": {"B": [14], "F": [56, 57, 58, 59]},
+                    "15": {"B": [15], "F": [60, 61, 62, 63]},
+                    "16": {"B": [16], "F": [64, 65, 66, 67]},
+                    "17": {"B": [17], "F": [68, 69, 70, 71]},
+                    "18": {"B": [18], "F": [72, 73, 74, 75]},
+                    "19": {"B": [19], "F": [76, 77, 78, 79]},
+                    "20": {"B": [20], "F": [80, 81, 82, 83]},
+                    "21": {"B": [21], "F": [84, 85, 86, 87]},
+                    "22": {"B": [22], "F": [88, 89, 90, 91]},
+                    "23": {"B": [23], "F": [92, 93, 94, 95]},
+                    "24": {"B": [24], "F": [96, 97, 98, 99]},
+                    "25": {"B": [25], "F": [100, 101, 102, 103]},
+                    "26": {"B": [26], "F": [104, 105, 106, 107]},
+                    "27": {"B": [27], "F": [108, 109, 110, 111]},
+                    "28": {"B": [28], "F": [112, 113, 114, 115]},
+                    "29": {"B": [29], "F": [116, 117, 118, 119]},
+                    "30": {"B": [30], "F": [120, 121, 122, 123]},
+                    "31": {"B": [31], "F": [124, 125, 126, 127]},
+                    "32": {"B": [32], "F": [128, 129, 130, 131]},
+                    "33": {"B": [33], "F": [132, 133, 134, 135]},
+                    "34": {"B": [34], "F": [136, 137, 138, 139]},
+                    "35": {"B": [35], "F": [140, 141, 142, 143]},
+                    "36": {"B": [36], "F": [144, 145, 146, 147]},
+                    "37": {"B": [37], "F": [148, 149, 150, 151]},
+                    "38": {"B": [38], "F": [152, 153, 154, 155]},
+                    "39": {"B": [39], "F": [156, 157, 158, 159]},
+                    "40": {"B": [40], "F": [160, 161, 162, 163]},
+                    "41": {"B": [41], "F": [164, 165, 166, 167]},
+                    "42": {"B": [42], "F": [168, 169, 170, 171]},
+                    "43": {"B": [43], "F": [172, 173, 174, 175]},
+                    "44": {"B": [44], "F": [176, 177, 178, 179]},
+                    "45": {"B": [45], "F": [180, 181, 182, 183]},
+                    "46": {"B": [46], "F": [184, 185, 186, 187]},
+                    "47": {"B": [47], "F": [188, 189, 190, 191]},
+                    "48": {"B": [48], "F": [192, 193, 194, 195]},
+                    "49": {"B": [49], "F": [196, 197, 198, 199]},
+                },
+            },
+        }
```

Comment:
Please use a static file for this data

Reply:
Until I have better data I would actually like the remove this completely. I think for now i will do this and create some more exact BMIM data for the test